### ISSUE TITLE
Webroot detecting malware - Win32.LocalInfect.2

### ISSUE THREAD

**Issue by MarquisDS** [cheat-engine/cheat-engine#2464]
I was recently referred to this program through Steelovsky's Youtube video on how to hack Elden Ring to make characters for PvP. When trying to download and run the game, Webroot stopped me every step of the way. First it pointed to something called "hacktool" which I assumed was part of the program. Then it pointed out the supposed malware group in the title, which I can't imagine would be part of the cheat engine. I'm worried that someone might have planted a virus into the program knowing people would flock to it after seeing Steelovsky's video. Can someone just confirm whether or not this is meant to be here so I can turn Webroot off without fear of infecting my computer?
![CheatEngineError](https://user-images.githubusercontent.com/123203767/213787509-626e837f-2135-4736-a377-69cd232360d5.PNG)

**Comment by cheat-engine** [comment]
never heard of that one.  looks like a cached file in chrome. likely a residue of when you downloaded CE

**Comment by MarquisDS** [comment]
You're right, it was a cached file. Webroot has quarantined it - actually, there were several from my several attempts to download it. Unfortunately I had Webroot delete most of them, and I tried downloading CE again just now and did not get the same warning. Here are some screenshots I was able to grab from the Properties tab of the remaining cached file.
![WeirdCacheThing1](https://user-images.githubusercontent.com/123203767/213820357-dd4b57a3-ebd9-4c1d-aac1-7dc5f17b3f06.png)
![WeirdCacheThing2](https://user-images.githubusercontent.com/123203767/213820358-1a1b7000-fc5f-43d8-808e-66712cd92d94.png)
